### PR TITLE
Update the QGIS server documentation

### PR DESCRIPTION
**Pull request by sbrunner**
Some documentation for #6047

### CODE REVIEW

File: doc/integrator/backend_qgis.rst [hunk 1]
```diff
@@ -54,16 +54,8 @@ API.
...
-
-.. code::
-
-    <WMSUrl type="QString">https://<host>/mapserv_proxy?ogcserver=<name></WMSUrl>
-    <WFSUrl type="QString">https://<host>/mapserv_proxy?ogcserver=<name></WFSUrl>
-    <WMTSUrl type="QString">https://<host>/mapserv_proxy?ogcserver=<name></WMTSUrl>
-    <WCSUrl type="QString">https://<host>/mapserv_proxy?ogcserver=<name></WCSUrl>
-
-By default, ``https://<host>/mapserv_proxy?ogcserver=<name>`` is empty.
+Finally, in project settings, in QGIS server section set all the published URL to:
```
Comment (jwkaltz) [inline doc/integrator/backend_qgis.rst]: Finally, in the project settings, in the QGIS server section, set all the published URL to:

File: doc/integrator/backend_qgis.rst [hunk 2]
```diff
@@ -76,10 +68,17 @@ and add a section for the DB you want to access:
 
    [<package>]
    host=<host>  # Localhost address of the Docker network interface (=> ${IP})
-   dbname=<database name>
+   port=5432
```
Comment (jwkaltz) [inline doc/integrator/backend_qgis.rst]: why not write
<database port>
?

File: doc/integrator/backend_qgis.rst [hunk 3]
```diff
@@ -95,37 +94,38 @@ to change the host parameter.
 
 Then, in QGIS, fill only the ``name`` and ``service`` fields when you create the DB connection.
 
+Use ``<package>`` for read only layers, and ``<package>_master`` If you need to edit the layer.
```
Comment (jwkaltz) [inline doc/integrator/backend_qgis.rst]: If -> if

you need to edit the layer -> the layer should be writable.

File: doc/integrator/backend_qgis.rst (hunk 2 above)
Comment (sbrunner) [reply doc/integrator/backend_qgis.rst]: ? what's the relation with write?

Comment (jwkaltz) [reply doc/integrator/backend_qgis.rst]: Hmm something disappeared in my comment.
I tried to say: 

why not write
  database port
instead of 5432.

Comment (jwkaltz) [reply doc/integrator/backend_qgis.rst]: It's funny, if you edit my earlier comment, you see what I wrote. But GitHub does not display it. I guess the smaller than and greater than characters are somehow special in comments.

Comment (sbrunner) [reply doc/integrator/backend_qgis.rst]: Effectively I can change that :-)

**Pull request merged**